content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -59,17 +59,17 @@ One of the [recommended labels](/docs/concepts/overview/working-with-objects/com
 
 Example: `app.kubernetes.io/name=mysql`
 
-Used on: All Objects
+Used on: All Objects (typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/)).
 
-The name of the application.
+The name of the application. Note that this label is typically used on [Workload Resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/).
```

Comment:
Use simple and direct language, and use consistent text and formatting with other examples of this language.

```suggestion
The name of the application (typically used on [workload resources](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/).
```